lib/kcerts/ssh_linux.go

```diff
@@ -0,0 +1,9 @@
+// +build linux
```

Comment:
Special comment to tell the golang compiler to only include this file for linux platforms.